spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/Tokenizer.java

```diff
@@ -47,7 +47,8 @@
   public static final String PLUGIN_NAME = "Tokenizer";
   private Config config;
   private Schema outputSchema;
-  private RegexTokenizer tokenizer;
+  // Work around because of HYDRATOR-1116. Remove this change once its fixed
+  private Object tokenizer;
```

Comment:
Seems like this doesn't need to be a field at all. Just new it in the RDD `transform` function.